std/typecons.d

```diff
@@ -734,6 +735,8 @@ if (distinctFieldNames!(Specs))
             Tuple!(int, int) t = ints;
         }
 
+        private enum noMemberHasCopyCtor(T) = !anySatisfy!(hasCopyCtor, Types);
```

Comment:
The `T` param is unused. Is it a (nested) template to evaluate it lazily only?

Reply:
LOL. Good catch. No, it should be lazy, but it wasn't written this way to be lazy. It was likely written this way, because I wrote the code after spending all day trying to figure out what the problem was, and I was tired, but since it worked, I didn't catch the mistake.

I've now moved the trait outside of `Tuple` so that it doesn't need a dummy argument to be lazy.